nomad/drainer/drainer.go

```diff
@@ -402,6 +402,7 @@ func (n *NodeDrainer) drainAllocs(future *structs.BatchFuture, allocs []*structs
 	}
 
 	evals := make([]*structs.Evaluation, 0, len(jobs))
+	now := time.Now().UTC().UnixNano()
```

Comment:
Double checking that we want all these evals to have the same timestamp. I would assume so since they're all part of the same event.

Reply:
Yes. You have the right idea: they're created and applied in a logical batch so they can share a timestamp.